### PR TITLE
feat(spanner): Handle null expiration time in the St…

### PR DESCRIPTION
**Pull request by satyaprakashg**
…artBackupOperation API

Context: Today, cloud customers have to mandatorily specify the expiration timestamp at the time of create/copy backup. With the default retention period feature, the expiration timestamp is made optional at the time of create/copy backup. If the customer doesn't specify the expiration timestamp, the default retention period of the backup is set to 1 year.

In this commit, making changes to handle null expiration time in the StartBackupOperation API. Do not set the expiration timestamp in the protobuf if the expire time passed in is zero.

### CODE REVIEW

**Review by rahul2393 — APPROVED** [review]
LGTM, please fix the vet issues.

**Comment by satyaprakashg** [comment]
Thank you. Sure, I will fix them. Can you please add "do not merge" label on this diff because I want to merge this after the backend changes are landed.

**Comment by codyoss** [comment]
Closing due to lack of response. Feel free to reopen if you have time to work on this again.

**Pull request closed without merging**
